testing/firefox/APKBUILD

```diff
@@ -100,7 +110,7 @@ build() {
 		--prefix=/usr \
 		\
 		--disable-crashreporter \
-		--disable-elf-hack \
+		$_disable-elf-hack \
```

Comment:
It took me some time to spot the difference: it's underscores instead of dashes.
`_disable_elf_hack`
`_disable-elf-hack`